server/allergenMaster.js

```diff
@@ -20,17 +21,21 @@ async function getMenuImage(body) {
+  const prompt = `In a JSON object response, provide seperate FOOD MENU ITEMS based on the following menu string  ${query}. This is from an OCR reading, do your best to extract actual menu items from this OCR string. Do not categorise the items at all, and ignore all other information. The different menu items should be under "menu_items". Do not include your answer inside a string, just a JSON Code block. Important points to consider in your response: 1. Cross-reference the spelling of menu items 2.Ensure there are no unnecessary special characters are not present in your response. 3. Remove menu items that do not seem like they are menu items. Such as names & singular numbers. 4. Ensure consistent capitalisation, new words will always start with a capital letter, and lowercase for the rest, such as "Chicken Pot Pie". 5. Structure the response like this {menu_items: ['Menu item 1', 'Menu Item 2', ...]}`;
+  await new Promise(resolve => setTimeout(resolve, 5000));
+  // return callGeminiJSON(prompt);
+  return callGPTJSON(prompt);
 }
 
 // Takes in a query (menu) and returns a JSON Object that contains eaach ingredient and its associated allergens.
 async function getIngredientDetails(query) {
   const dietaryRequirements =
   "ALCOHOL,CELERY,CRUSTACEAN,DAIRY,DASH,EGG,FISH,FODMAP,GLUTEN,IMMUNO_SUPPORTIVE,NON_KETO_FRIENDLY,NON_KIDNEY_FRIENDLY,KOSHER,HIGH_POTASSIUM,HIGH_SUGAR,LUPINE,MEDITERRANEAN,MOLLUSK,MUSTARD,OIL_ADDED,NON_PALEO_FRIENDLY,PEANUT,NON_PESCATARIAN,PORK,RED_MEAT,SESAME,SHELLFISH,SOY,SULPHITE,TREE_NUT,NON_VEGAN,NON_VEGETARIAN,WHEAT";
 
-  const prompt = `In only a JSON object response, provide the ingredients for ${query}, inside each ingredient, map a list of allergens to each ingredient. Given these allergens come from the following list ${dietaryRequirements}. Do not include your answer inside a string, just a JSON Code block. Be very accurate with the ingredients you list, ensuring to take inspiration from the dish name. Do not list the amount of ingredients, simply identify them. Structure the JSON so that it is exactly like {ingredients: [{name:"", allergens:[]}, and so on]} RETURN A JSON AND PUT ALL THIS UNDER INGREDIENTS:`;
+  const prompt = `In only a JSON object response, provide the ingredients for ${query}, inside each ingredient, map a list of allergens to each ingredient. Given these allergens come from the following list ${dietaryRequirements}. Do not include your answer inside a string, just a JSON Code block. Be very accurate with the ingredients you list, ensuring to take inspiration from the dish name. Do not list the amount of ingredients, simply identify them. Do not include more than 25 ingredients per meal. Structure the JSON so that it is exactly like {ingredients: [{name:"", allergens:[]}, and so on]} RETURN A JSON AND PUT ALL THIS UNDER INGREDIENTS:`;
+  await new Promise(resolve => setTimeout(resolve, 10000));
   return callGeminiJSON(prompt);
+  // return callGPTJSON(prompt);
```

Comment:
Why is this return commented out if we're going with the old one?

Reply:
idk

Reply:
For quick switching between our two models! 😁

Reply:
Why did you bother to comment in the first place